druid/src/widget/image.rs

```diff
@@ -25,7 +25,52 @@ use crate::{
     PaintCtx, Rect, RenderContext, Size, UpdateCtx, Widget,
 };
 
-/// A widget that renders an Image
+/// A widget that renders a bitmap Image.
+///
+/// Contains data about how to fill given space and interpolate pixels.
+/// Configuration options are provided via the builder pattern.
+///
+/// Note: interpolation can lead to blurry or pixelated images and so is not
+/// recommended for things like icons. Instead consider using
+/// [SVG files](https://en.wikipedia.org/wiki/Scalable_Vector_Graphics)
+/// and enabling the `svg` feature with `cargo`.
+///
+/// (See also:
+/// [`ImageData`],
+/// [`FillStrat`],
+/// [`InterpolationMode`]
+/// )
+///
+/// # Example
+///
+/// Create an image widget and configure it using builder methods
+/// ```
+/// use druid::{
+///     widget::{Image, ImageData, FillStrat},
+///     piet::InterpolationMode,
+/// };
+///
+/// let image_data = ImageData::empty();
+/// let image_widget = Image::new(image_data)
+///     // set fill strategy
+///     .fill_mode(FillStrat::Fill)
+///     // set interpolation mode
+///     .interpolation_mode(InterpolationMode::NearestNeighbor);
```

Comment:
Not sure that we want to show the example with nearest neighbor. That's rarely what is desired.

Reply:
right. it's part of my particular use case (texture packer) so maybe i was a little too fixated on it.

easily fixed!